Extension/package.nls.json

```diff
@@ -43,7 +43,7 @@
     "c_cpp.configuration.configurationWarnings.description": "Determines whether pop up notifications will be shown when a configuration provider extension is unable to provide a configuration for a source file.",
     "c_cpp.configuration.intelliSenseCachePath.description": "Defines the folder path for cached precompiled headers used by IntelliSense. The default cache path is \"%LocalAppData%/Microsoft/vscode-cpptools\" on Windows, \"$XDG_CACHE_HOME/vscode-cpptools/\" on Linux (or \"$HOME/.cache/vscode-cpptools/\" if XDG_CACHE_HOME is not defined), and \"$HOME/Library/Caches/vscode-cpptools/\" on Mac. The default path will be used if no path is specified or if a specified path is invalid.",
     "c_cpp.configuration.intelliSenseCacheSize.description": "Maximum size of the per-workspace hard drive space in megabytes for cached precompiled headers; the actual usage may fluctuate around this value. The default size is 5120 MB. Precompiled header caching is disabled when the size is 0.",
-    "c_cpp.configuration.default.includePath.description": "The value to use in a configuration if \"includePath\" is not specified, or the values to insert if \"${default}\" is present in \"includePath\".",
+    "c_cpp.configuration.default.includePath.description": "The value to use in a configuration if \"includePath\" is not specified in c_cpp_properties.json. If such includePath exists, add \"${default}\" to includePath to cause \"default.includePath\" in settings to be added and used.",
```

Comment:
I don't think this text should change.

Reply:
The last decision was to modify this text for this scenario:
When the user already has a "c_cpp_properties.json", and then tries to change the "C_Cpp.default.includePath" in "settings.json". 

there was a second scenario in which the user doesn't have a "c_cpp_properties.json", and has set the the "C_Cpp.default.includePath" in "settings.json". In this scenario, we added the default automatically. 

The users' issue was with the first scenario, which we only changed the text for providing a hint fro them.

Reply:
`The value to use in a configuration if \"includePath\" is not specified in c_cpp_properties.json. If \"includePath\" is specified, add \"${default}\" to the array to insert the values from this setting.`